File: src/Application Specific/Forms And Controls/FleetVan/FRMAddVanEquipment.cs
using System;
using System.Windows.Forms;
using Microsoft.VisualBasic.CompilerServices;

namespace FSM
{
    public partial class FRMAddVanEquipment
    {
        public FRMAddVanEquipment()
        {
            InitializeComponent();
        }

        private void btnAdd_Click(object sender, EventArgs e)
        {
            if (Conversions.ToDouble(Combo.get_GetSelectedItemValue(cboEquipment)) == 0)
            {
                App.ShowMessage("Please select equipment", MessageBoxButtons.OK, MessageBoxIcon.Information);
            }
            else if (Modal)
            {
                Close();
            }
            else
            {
                Dispose();
            }
        }

        private void FRMAddVanEquipment_Load(object sender, EventArgs e)
        {
            var argc = cboEquipment;
            Combo.SetUpCombo(ref argc, App.DB.FleetEquipment.GetAll().Table, "EquipmentID", "Name", Entity.Sys.Enums.ComboValues.Please_Select);
        }
    }
}

File: src/Application Specific/Forms And Controls/FleetVan/FRMAddVanEquipment.vb
Public Class FRMAddVanEquipment

    Private Sub btnAdd_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnAdd.Click
        If Combo.GetSelectedItemValue(cboEquipment) = 0 Then
            ShowMessage("Please select equipment", MessageBoxButtons.OK, MessageBoxIcon.Information)
        Else
            If Me.Modal Then
                Me.Close()
            Else
                Me.Dispose()
            End If
        End If
    End Sub

    Private Sub FRMAddVanEquipment_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Combo.SetUpCombo(cboEquipment, DB.FleetEquipment.GetAll.Table, "EquipmentID", "Name", Entity.Sys.Enums.ComboValues.Please_Select)
    End Sub

End Class